### DISCUSSION TITLE
upgrade.sig File Empty

### DISCUSSION THREAD

**Discussion by leegengyu** [rickysarraf/apt-offline#205] [Q&A]
Hi,

I am using version `1.8.5`.

I ran these commands:

1. (Offline machine) `sudo apt-offline set update.sig --update`
2. (Online machine) `sudo apt-offline get update.sig --bundle update.zip`
3. (Offline machine) `printf 'y\n' | sudo apt-offline install update.zip --allow-unauthenticated`
4. (Offline machine) `sudo apt-offline set upgrade.sig --upgrade`

After running the 4th command, my `upgrade.sig` is empty, despite no errors in any of the above commands' output.

Thank you!

**Comment by rickysarraf** [comment]
I need the screen output of step no. 3. It seems your `install` operation did not succeed.
